fedora/Dockerfile

```diff
@@ -1,13 +1,9 @@
-FROM fedora:20
+FROM fedora
 MAINTAINER amitsaha.in@gmail.com
 
-# Let's start with some basic stuff.
-RUN yum -y clean all
-RUN yum -y update
-RUN yum install -y iptables ca-certificates lxc e2fsprogs
-
-# Install Docker from Fedora repos
-RUN yum -y install docker-io
+# Update and install docker
+RUN dnf -y update && dnf clean all
+RUN dnf -y install docker && dnf clean all
```

Comment:
I think these two `RUN` statements should be combined.

In addition; should we install the Fedora packaged docker here, or the one packaged by Docker itself? Docker recently set up their own apt/yum repos (see https://github.com/docker/docker/pull/13680). Instructions for installing from those repos can be found here;
https://github.com/docker/docker/blob/master/docs/installation/fedora.md

(I agree with using `dnf` here, although `yum` still works (it's an alias for `dnf`).

@jpetazzo any thoughts on this? ("fedora" vs "docker" managed repo?)